src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -808,18 +808,23 @@ internal DefinitionResult GetDefinition(TextDocumentPosition textDocumentPositio
             {
                 try
                 {
+                    // Handle tokenizing for Peek Definition 
+                    Position tokenPosition = new Position();
```

Comment:
We have an internal class called BufferPosition, which tends to use the VSCode Position values but 1-indexed. Perhaps for clarity, we should change the code to use that? Otherwise it's kind of confusing since we're adjusting this but nobody would know why.